package.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "smart-underline",
   "version": "v4.2.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/EagerIO/SmartUnderline.git"
+  },
+  "keywords": [
+    "smart",
+    "underline",
+    "Readable",
```

Comment:
If you want me to change, I'll do, personanlly both fine, what do you think? Thanks.